File: examples/node/printAlphabet.js
// Print uppercase and lowercase letters for a language code and script.
const { getUppercase, getLowercase } = require('../..');

async function main() {
  const code = process.argv[2] || 'en';
  const script = process.argv[3];
  const upper = await getUppercase(code, script);
  const lower = await getLowercase(code, script);
  const label = script ? `${code}-${script}` : code;
  console.log(`${label} uppercase: ${upper.join(' ')}`);
  console.log(`${label} lowercase: ${lower.join(' ')}`);
}

main();

File: examples/python/print_alphabet.py
"""Print uppercase and lowercase letters for a language code."""
from __future__ import annotations

import sys

from worldalphabets import load_alphabet


def main() -> None:
    code = sys.argv[1] if len(sys.argv) > 1 else "en"
    script = sys.argv[2] if len(sys.argv) > 2 else None
    alphabet = load_alphabet(code, script=script)
    label = f"{code}-{script}" if script else code
    print(f"{label} uppercase: {' '.join(alphabet.uppercase)}")
    print(f"{label} lowercase: {' '.join(alphabet.lowercase)}")


if __name__ == "__main__":
    main()
